Upstream-Status: Inappropriate [configuration]

without the patch, /usr/include/rpm/ won't be added
in cppflags

8/19/2010 - created by Qing He <qing.he@intel.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b040a6..04a1595 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,6 +21,10 @@ SET( CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -g -O2 -Wall -fstack-protector" )
 
 GENERATE_PACKAGING(${PACKAGE} ${VERSION})
 
+FIND_PACKAGE(PkgConfig REQUIRED)
+PKG_CHECK_MODULES(RPM REQUIRED rpm)
+INCLUDE_DIRECTORIES( ${RPM_INCLUDE_DIRS} )
+
 FIND_PACKAGE( Zypp REQUIRED )
 IF( ZYPP_FOUND )
   INCLUDE_DIRECTORIES(${ZYPP_INCLUDE_DIR})
